hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -688,6 +692,17 @@ public AbfsRestOperation append(final String path, final byte[] buffer,
     try {
       op.execute(tracingContext);
     } catch (AzureBlobFileSystemException e) {
+      /*
+        If the http response code indicates a user error we retry the same append request with expect header disabled.
```

Comment:
thanks for adding comments. Use the // style of comments as written for method appendSuccessCheckOp() below. 
Also the line lengths is too long I guess, looks off as compared to current codebase. please limit it to 100.

Reply:
Made the changes.